src/cli/src/topic/create.rs

```diff
@@ -33,7 +33,6 @@ pub struct CreateTopicOpt {
         long = "partitions",
         value_name = "partitions",
         default_value = "1",
-        required_unless = "replica_assignment"
```

Comment:
why was this line removed?

Reply:
The lines were removed to remove  the optional parameters `--partitions` and `--replication` from the below command
`fluvio topic create --help`